### DISCUSSION TITLE
Assuming A Role/Default Session Duration

### DISCUSSION THREAD

**Discussion by jacobzlogar** [aws-actions/configure-aws-credentials#635] [General]
[docs](https://github.com/aws-actions/configure-aws-credentials#assuming-a-role)
- this action tries to request a 6 hour session duration when assuming a role: i believe roles are created with a default 4 hour session duration (through the ui at least) which can be kind of confusing - might want to add a blurb explaining this?

**Comment by peterwoodworth** [comment]
Could you further describe this default 4 hour session duration and the confusion you experienced?  I believe the default in most cases is 1 hour, though the default that we set shouldn't conflict with that. 

We already have an issue with our plan for this described: https://github.com/aws-actions/configure-aws-credentials/issues/103. We will be adjusting the default to be 1 hour in all cases in our next major version release, but cannot further change the defaults in the current releases in case this change would break existing users.
